### ISSUE TITLE
train work

### ISSUE THREAD

**Issue by hylrh2008** [xuduo35/unet_mxnet2ncnn#3]
when i train the unet i get this loss value:
Epoch 8: Moving Training Loss -0.98323, Validation Loss -0.94432

raining 0/10 ./datasets/humanparsing/JPEGImages/2500_1005.jpg
raining 1/10 ./datasets/humanparsing/JPEGImages/2500_1004.jpg
raining 2/10 ./datasets/humanparsing/JPEGImages/2500_10.jpg
raining 3/10 ./datasets/humanparsing/JPEGImages/2500_1002.jpg
raining 4/10 ./datasets/humanparsing/JPEGImages/2500_1006.jpg
raining 5/10 ./datasets/humanparsing/JPEGImages/2500_1003.jpg
raining 6/10 ./datasets/humanparsing/JPEGImages/2500_1000.jpg
raining 7/10 ./datasets/humanparsing/JPEGImages/2500_100.jpg
raining 8/10 ./datasets/humanparsing/JPEGImages/2500_1001.jpg
raining 9/10 ./datasets/humanparsing/JPEGImages/2500_1.jpg
raining 10/10 
g 0/10 ./datasets/humanparsing/JPEGImages/dataset10k_6506.jpg
g 1/10 ./datasets/humanparsing/JPEGImages/dataset10k_6507.jpg
g 2/10 ./datasets/humanparsing/JPEGImages/dataset10k_6508.jpg
g 3/10 ./datasets/humanparsing/JPEGImages/dataset10k_6509.jpg
g 4/10 ./datasets/humanparsing/JPEGImages/dataset10k_651.jpg
g 5/10 ./datasets/humanparsing/JPEGImages/dataset10k_6510.jpg
g 6/10 ./datasets/humanparsing/JPEGImages/dataset10k_6511.jpg
g 7/10 ./datasets/humanparsing/JPEGImages/dataset10k_6512.jpg
g 8/10 ./datasets/humanparsing/JPEGImages/dataset10k_6513.jpg
g 9/10 ./datasets/humanparsing/JPEGImages/dataset10k_6514.jpg
g 10/10 
Epoch 9: Moving Training Loss -0.98339, Validation Loss -0.94633

is this loss value right? thank you!

**Comment by xuduo35** [comment]
yes. you can check the loss function. i use code from other project.

**Comment by hylrh2008** [comment]
thanks! i changed the loss function got the positive loss value
